### PR TITLE
(#44) MongoDB reconnect

### LINKED ISSUE #44: Bot does not reconnect if initial MongoDB connection failed
## Expected Behavior

Bot reconnecting

## Current Behavior

Not reconnecting

## Possible Solution

Error handling and reconnect

## Steps to Reproduce (for bugs)

1. Stop MongoDB and Bot container
2. Start bot container
3. Start MongoDB container

## Context

MongoDB is required for bot to work properly

### PR DESCRIPTION
**Pull request by mkevenaar**
Reconnect

## Description

The bot will reconnect to the database every 5000ms if the initial connection failed. 

## Related Issue

Fixes #44 

## Motivation and Context

Issue #44

## How Has This Been Tested?

By using an invalid mongodb connection string, bot tries to reconnect

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/bot.js [hunk 1]
```diff
@@ -117,3 +106,19 @@ export async function initBot() {
...
 }
+
+// Connect to the database
+async function connectWithRetry(MONGODB) {
+  return mongoose
+    .connect(MONGODB, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+    })
+    .then((_) => {
```
Comment (Lucxjo) [inline src/bot.js:117]: Do you need the underscore here? You could get away with omitting it.

**Review by Lucxjo — COMMENTED** [review]
All looks good, could you just clarify something for me please?

File: src/bot.js (hunk 1 above)
Comment (mkevenaar) [reply src/bot.js:117]: The _ was already there. But I guess you are right

Comment (mkevenaar) [reply src/bot.js]: Fixed

**Review by Lucxjo — APPROVED** [review]
Looks good!

**Pull request merged**